test/al_aws_collector_test.js

```diff
@@ -744,6 +746,36 @@ describe('al_aws_collector tests', function() {
             collector.done(stringifialbleError);
         });
 
+        it('calls success when there is no error and stremtype is passed', (done) => {
```

Comment:
please add a test for a collector with streams

Reply:
sure